arrow-fx/src/main/kotlin/arrow/fx/internal/UncancellableMVar.kt

```diff
@@ -135,7 +135,7 @@ internal class UncancellableMVar<F, A> private constructor(initial: State<A>, pr
         if (queue.isEmpty()) {
           if (stateRef.compareAndSet(current, State.empty())) {
             // Signals completion of `take`
-            onTake(Either.Right(value))
+            onTake(Right(value))
```

Comment:
`Either.Right` is the constructor, `Right` goes through an additional method for no reason here.

Reply:
Indeed, it actually goes through 2 additional functions :/